packages/realm/src/app-services/SyncSession.ts

```diff
@@ -305,7 +305,7 @@ export class SyncSession {
    */
   get config(): SyncConfiguration {
     return this.withInternal((internal) => {
-      const user = new User(internal.user, mockApp);
+      const user = new User(internal.config.user, mockApp);
```

Comment:
`User` currently takes a `binding.User` as 1st arg, but `internal.config.user` returns a `binding.SyncUser`, so this won't transpile. Hopefully this is an easy fix 🤞